SM/internal/utils/handler_utils/output/convertOutputData.go

```diff
@@ -0,0 +1,98 @@
+package handler_output
+
+import (
+	"errors"
+	"sm/internal/database/postgres"
+)
+
+func ConvertToOut(dest interface{}) (interface{}, error) {
```

Comment:
Signature and naming does not have semantics - think why do you need it, at least add some comments